cfpb/retirement
# Modernize build | Update cf-icons to v10 | remove some jquery

## Removals

- Removes some jquery references.
- Removes document read jquery, since JS is loaded below the main content.
- Removes some unused cf-icon CSS.

## Changes

- Update babel to latest.
- Update cf-icons to 10.0.0 so remove unused reference to webfont icons that was breaking collectstatic.
- Update del, gulp, gulp-changed, gulp-sourcemaps, protractor, require-dir, snyk, through2.
- Update to ES6 module default exports.
- Fixes some general linter warnings (line length, import order, const variables where let was used).

## Testing

To test inside cfgov-refresh
- With a recent db load, activate your refresh env, pull in this branch locally (in this example, retirement is cloned to /develop-apps/ ), and then install the branch. 
```bash
pip install -e ./develop-apps/retirement/
```
- cd to the cfgov-refresh root, run the migrations and load the new content:
```bash
./cfgov/manage.py migrate
./cfgov/manage.py loaddata retiredata.json
```
- fire up `./runserver.sh` and check that the tool works like prod:
<http://localhost:8000/consumer-tools/retirement/before-you-claim/>
